### ISSUE TITLE
Training from scratch VS finetune performance

### ISSUE THREAD

**Issue by dabasmoti** [flairNLP/flair#1855]
Hi,
I am traning  a language model from scratch and finetune one.

the ms/batch on the same data is extremely different

FINETUNE: 703 ms
`2020-09-08 07:52:09,181 | split   4 / 10 |  6500/12658 batches | ms/batch 703.28 | loss  1.25 | ppl     3.48`

FROM SCRATCH: 163 ms
`2020-09-08 07:55:08,167 | split   5 / 10 |  6500/12658 batches | ms/batch 163.23 | loss  2.87 | ppl     4.98
`
I am wonder why the difference is so huge?

I am running on same setup
16 vCPU, 60GB 
SSD
Nvidia p100, 

Second question:
Every time when i start to train it take more than 3 hours for the training to start because it preprocessing the texts file
I have 10 splits of  ~ 500 MB  each.
Why it take so long for the text preprocessing?

**Comment by alanakbik** [comment]
Hello @dabasmoti one reason I could think of is if the model you fine-tune uses a larger character dictionary than the model you train from scratch. Do the models have different character dictionaries?

**Comment by dabasmoti** [comment]
@alanakbik 
Hi,
Yes, the dictionaries are different
